ansible/artifacts/sunbird/login/login-reset-password.ftl

```diff
@@ -38,19 +36,17 @@
                             <#else>${msg("email")}
                             </#if>
                         </label>
-                        <input type="text" id="username" class="mt-8" name="username" onfocusin="inputBoxFocusIn(this)" onfocusout="inputBoxFocusOut(this)" autofocus/>
+                        <input type="text" id="username" class="mt-8" placeholder="Enter your email / mobile number" name="username" onfocusin="inputBoxFocusIn(this)" onfocusout="inputBoxFocusOut(this)" autofocus/>
                     </div>
                     <div class="field">
-                        <button id="login" class="ui fluid submit button">
+                        <button id="login" class="sb-btn sb-btn-normal sb-btn-primary width-100">
                         ${msg("doReset")}
                         </button>
                     </div>
                 </form>
-                <div class="${properties.kcFormOptionsWrapperClass!} signUpMsg mb-56 mt-45 textCenter">
-                    <span><a id="versionLink" class="backToLogin" onclick="javascript:makeDivUnclickable()" href="${url.loginUrl}"><span class="fs-14"><< </span> ${msg("backToLogin")}</a></span>
+                <div class="${properties.kcFormOptionsWrapperClass!} signUpMsg mb-56 mt-24 textCenter">
```

Comment:
please check that the class backToLogin is not used in the JS

Reply:
No